### PR TITLE
Fix minikube issues

### PR DESCRIPTION
**Pull request by edvald**
This addresses a few issues that have affected minikube users, but (for some reason, still investigating) not Docker for Desktop users.

### CODE REVIEW

File: garden-service/src/garden.ts [hunk 1]
```diff
@@ -441,15 +441,16 @@ export class Garden {
...
-    if (configureHandler) {
-      const configureOutput = await configureHandler({ config: providerConfig })
-      providerConfig = configureOutput.config
-    }
 
     if (plugin.configSchema) {
       providerConfig = validate(providerConfig, plugin.configSchema, { context: `${pluginName} configuration` })
     }
 
+    if (configureHandler) {
```
Comment (eysi09) [inline garden-service/src/garden.ts]: Not sure I understand the context here, why did you change the order?

**Comment by eysi09** [comment]
Should we also address the hardcoded `ingressClass` value:
https://github.com/garden-io/garden/blob/be5448381ed1a9c9e1b8808a3279204c8e54ecb2/garden-service/src/plugins/kubernetes/local.ts#L143
in this commit?

File: garden-service/src/garden.ts (hunk 1 above)
Comment (edvald) [reply garden-service/src/garden.ts]: Oh right, I should have explained. Thing is, the configSchema may specify default values, which are lost if the clauses are swapped back.

**Comment by edvald** [comment]
Yes, good catch. I'll add that.

**Comment by edvald** [comment]
@eysi09 added a commit to fix the hardcoded ingress class. Could you test this on your setup?

**Comment by eysi09** [comment]
Will do.

**Comment by eysi09** [comment]
Yes, toggling the `setupIngressController` now works.
